lib/restful_resource/base.rb

```diff
@@ -119,12 +121,18 @@ def self.collection_url(params)
       replace_parameters(url, params)
     end
 
+    def self.context_lambda
+      supperclass.instance_variable_get :@context_lambda
```

Comment:
superclass? I think this couls just be `@context_lambda`, no?

Reply:
e.g., there's no special stuff to make `@http` work.

Reply:
Sadly no =(.

Since this hierarchy has a particular way to configure it, the _lambda_ is stored at `Base`, class instance variable `@context_lambda`